tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -881,6 +886,172 @@ def _wait_for_traffic(self, producer, acked, timeout=1800):
                 f"Low throughput while preparing for the test: {_throughput} MB/s"
             ) from None
 
+    # returns a tuple of (replicas, readyReplicas)
+    def get_cluster_replicas(self, cluster_name: str) -> tuple[int, int]:
```

Comment:
cluster_name is ignored in the v2 case? this could use a doc

Reply:
This has been updated so that if the cluster name format changes again or for some future provider, only one location in the code needs to be changed. (operator V2 currently hardcodes the cluster/statefulset name to `redpanda-broker`, and this is now noted in the code for the new `self.cluster_name` property).